### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by hardikguptaofficialgit** [mansiruhil13/Bobble-AI#1190]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

The login and registrations system does not work properly so to make it properly 
Features of Login and Registration System :- Create a customize system for users to interact with website

### Add Screenshots

![image](https://github.com/user-attachments/assets/bb332cb3-b250-4df5-ba8d-adb9b0cd9720)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC contributor
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by hardikguptaofficialgit** [comment]
Assign me this task !

**Comment by mansiruhil13** [comment]
can you elaborate the changes??

**Comment by hardikguptaofficialgit** [comment]
User Management :- 
Registration and Login System: Secure user registration and login with options for social media and Google sign-in.

Admin Panel
Dashboard for Real-Time Monitoring: Admins can view real-time ambulance locations, requests, and track activity.
Driver and Hospital Management: Tools for managing ambulance drivers, vehicle availability, hospital profiles, and ensuring active status.
Analytics and Reporting: Insights on service usage, peak times, response rates, and user satisfaction.
User Panel:-
Profile Management: User profile page with details such as contact info, emergency contacts, preferred hospitals, and medical information .  
Basically i will deploy website on free subdomains like infinityfree.net so we can do and we can also do on other deploying sites as well
